docs/contributing_to_vhive.md

```diff
@@ -86,3 +86,19 @@ git rebase main
 
 ### Avoiding Merge Commits
 As mentioned throughout this document, merge commits should be avoided. By having a linear commit history the changes made to the project are more clean, readable, and structured, and avoiding merged pull requests also makes the PR easier to incorporate into the main branch. To avoid merge commits one should follow the guidelines described above, particularly taking care to rebase on the main branch and keeping your forks in sync.
+
+### Using Go modules with private Git repo

```

Comment:
Make a separate section (## level), called Recommendations, and enclose this subsection there because using private modules is not a requirement like the ones above